tests/integration/testdata/invoke/template-return-image.yaml

```diff
@@ -7,6 +7,6 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       Handler: main.image_handler
-      Runtime: python3.12
+      Runtime: python3.11
```

Comment:
Did we ever find a root cause for this?

Reply:
When we make the request to lambda it only ever returns a 502, provided the host machine is windows and the lambda is returning raw bytes. I assume that it's because a bug between docker and the python 3.12 runtime but I wasn't able to narrow it down further than that

Reply:
Could that be docker version limitation for these new (al2023 based) runtimes? Were you able to run the tests on a Windows machine with newer Docker version to see if these tests are passing or not?